src/Writer/Extension/PodcastIndex/Feed.php

```diff
@@ -106,6 +110,189 @@ public function setPodcastIndexFunding(array $value): Feed
+                'invalid parameter: key "description" of "updateFrequency" must be of type string'
+            );
+        }
+        if (isset($value['complete']) && ! is_bool($value['complete'])) {
+            throw new Writer\Exception\InvalidArgumentException(
+                'invalid parameter: key "complete" of "updateFrequency": must be of type boolean'
+            );
+        }
+        if (isset($value['dtstart']) && ! is_string($value['dtstart'])) {
+            throw new Writer\Exception\InvalidArgumentException(
+                'invalid parameter: key "dtstart" of "updateFrequency" must be an ISO8601-formatted string'
+            );
+        }
+        if (isset($value['rrule']) && ! is_string($value['rrule'])) {
+            throw new Writer\Exception\InvalidArgumentException(
+                'invalid parameter: key "rrule" of "updateFrequency" must be of type string'
+            );
+        }
+        $this->data['updateFrequency'] = $value;
+        return $this;
+    }
+
+    /**
+     * Add feed person
+     *
+     * @param array<string,mixed> $value [name: str, role: str|null, group: str|null, img: url|null, href: url|null]
+     * @return $this
+     * @throws Writer\Exception\InvalidArgumentException
+     */
+    public function addPodcastIndexPerson(array $value)
+    {
+        if (! isset($value['name'])) {
+            throw new Writer\Exception\InvalidArgumentException(
+                'invalid parameter: "person" must be an array containing at least the key "name"'
+            );
+        }
+        if (! is_string($value['name'])) {
+            throw new Writer\Exception\InvalidArgumentException(
+                'invalid parameter: key "name" of "person" must be of type string'
+            );
+        }
+        if (isset($value['img']) && ! filter_var($value['img'], FILTER_VALIDATE_URL)) {
+            throw new Writer\Exception\InvalidArgumentException(
+                'invalid parameter: key "img" of "person" must be a url, starting with "http://" or "https://"'
+            );
+        }
+        if (isset($value['href']) && ! filter_var($value['href'], FILTER_VALIDATE_URL)) {
+            throw new Writer\Exception\InvalidArgumentException(
+                'invalid parameter: key "href" of "person" must be a url, starting with "http://" or "https://"'
+            );
+        }
+        if (! isset($this->data['persons'])) {
+            $this->data['persons'] = [];
+        }
+
+        /** @var array<array<string, mixed>> $this->data['persons'] */
+        $this->data['persons'][] = $value;
+        return $this;
+    }
+
+    /**
+     * Set a new array of persons.
+     * If no argument is passed, it will just remove all existing persons.
+     *
+     * @param null|array<array> $values
+     * @return $this
+     * @throws Writer\Exception\InvalidArgumentException
+     */
+    public function setPodcastIndexPersons(?array $values = null)
```

Comment:
Why not:
```suggestion
    public function setPodcastIndexPersons(array $values = []): self
```

Also, why "Persons" as opposed to "People" - does this follow the naming in the podcast index spec?

Reply:
The term `person` is defined in the specification and it must also be used here: https://github.com/Podcastindex-org/podcast-namespace/blob/main/docs/tags/person.md

Reply:
In it's handling as single `person` entry, it is still used here as such. Nevertheless, for the handling of a set of multiple `person` entries, I followed your suggestion to use the more intuitive term `people`, since I did not see the PodcastIndex docs speaking against such an implementation.

Reply:
`people` is for me not _intuitive_ because in the specification it is called person and therefore I expected `persons` as a user – even if I am aware of the plural form `people`. Which means `persons` _is closer_ to the specification for me.

By the way the `persons` is also used in the Podcastindex-org: https://github.com/search?q=org%3APodcastindex-org%20persons&type=code

Reply:
@froschdesign  Ok, this is in fact crucial. May I add additional alias methods using the term `persons`?